03-Node-npm-modules/ver.0.2/downloableMovie v0.2.js

```diff
@@ -0,0 +1,107 @@
+var MovieObserver = function ( ) {};
```

Comment:
**WARNING:** FILE NAMES MUST BE IN LOSER CASE LIKE:

`downlodable-movie_v0.2.js`